packages/typescript-estree/src/parser.ts

```diff
@@ -30,12 +30,12 @@ const log = debug('typescript-eslint:typescript-estree:parser');
  * This needs to be kept in sync with the top-level README.md in the
  * typescript-eslint monorepo
  */
-const SUPPORTED_TYPESCRIPT_VERSIONS = '>=3.3.1 <4.5.0';
+const SUPPORTED_TYPESCRIPT_VERSIONS = '>=3.3.1 <4.6.0';
 /*
  * The semver package will ignore prerelease ranges, and we don't want to explicitly document every one
  * List them all separately here, so we can automatically create the full string
  */
-const SUPPORTED_PRERELEASE_RANGES: string[] = ['4.4.0-beta', '4.4.1-rc'];
+const SUPPORTED_PRERELEASE_RANGES: string[] = ['4.5.0-beta'];
```

Comment:
4.5rc is out so we should add this

```suggestion
const SUPPORTED_PRERELEASE_RANGES: string[] = ['4.5.0-beta', '4.5.1-rc'];
```